sample.txt

```diff
@@ -1,5 +1,6 @@
-This is sample text.
+This is somple text.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/litmon/danger-sample/blob/770da37c1fde0bf856172054dbd32e7d1d50c89a/sample.txt#L1"></span>It is typo</td>
    </tr>
  </tbody>
</table>